package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "docs",
+  "scripts": {
+    "lint": "yarn install && yarn lint-md && yarn lint-js && yarn lint-php && yarn lint-language",
+    "lint-fix": "yarn install && yarn lint-md --fix; yarn lint-js --fix; yarn lint-php --fix; echo 'Fixed all auto-fixable problems'",
+    "lint-md": "yarn install && markdownlint-cli2 en/**/*.md",
+    "lint-js": "yarn install && eslint en",
+    "lint-php": "composer install --prefer-dist --no-progress --ansi --no-interaction --no-scripts --no-plugins --optimize-autoloader && vendor/bin/mdphpcs en",
+    "lint-language": "yarn install && vale sync && vale en"
```

Comment:
```suggestion
    "lint-language": "vale sync && vale en"
```

Reply:
What does `vale sync` do?

Reply:
Marking resolved as this is now out of scope